### PR TITLE
[V3-1315] link bug fix

### PR DESCRIPTION
**Pull request by NikolaiYurchenko**

### CODE REVIEW

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 1]
```diff
@@ -59,7 +59,7 @@
                     <path d="M18.1489 17.043H21.9149V28H18.1489V17.043ZM20 12C20.5816 12 21.0567 12.1823 21.4255 12.5468C21.8085 12.8979 22 13.357 22 13.9241C22 14.4776 21.8085 14.9367 21.4255 15.3013C21.0567 15.6658 20.5816 15.8481 20 15.8481C19.4184 15.8481 18.9362 15.6658 18.5532 15.3013C18.1844 14.9367 18 14.4776 18 13.9241C18 13.357 18.1844 12.8979 18.5532 12.5468C18.9362 12.1823 19.4184 12 20 12Z" fill="#F5F6FA"/>
                 </svg>
                 <div class="notification-wrap__text">
-                    <p>If the team member you want to invite to join the project is still not on this Satellite, please share this link to the signup page and ask them to register here: <a>www.storj.io/satellite/register</a></p>
+                    <p>If the team member you want to invite to join the project is still not on this Satellite, please share this link to the signup page and ask them to register here: <a href="https://storj.io/">www.storj.io/satellite/register</a></p>
```
Comment (egonelbre) [inline web/satellite/src/components/team/AddUserPopup.vue]: Shouldn't this link be relative to the site serving the page?

Comment (stefanbenten) [reply web/satellite/src/components/team/AddUserPopup.vue]: Yeah, i agree.
Besides that www.storj.io doesnt exists (or rather the redirect isnt working right now)

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 2]
```diff
@@ -294,6 +302,12 @@ export default class AddUserPopup extends Vue {}
         font-family: 'montserrat_regular' !important;
         font-size: 16px;
         line-height: 25px;
+
+        a {
+            color: #2683FF;
+            text-decoration: none;
```
Comment (egonelbre) [inline web/satellite/src/components/team/AddUserPopup.vue]: Do not remove underline decoration from links, unless it's an actual button. It's an important signifier for color-blind people.

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 3]
```diff
@@ -59,7 +59,7 @@
                     <path d="M18.1489 17.043H21.9149V28H18.1489V17.043ZM20 12C20.5816 12 21.0567 12.1823 21.4255 12.5468C21.8085 12.8979 22 13.357 22 13.9241C22 14.4776 21.8085 14.9367 21.4255 15.3013C21.0567 15.6658 20.5816 15.8481 20 15.8481C19.4184 15.8481 18.9362 15.6658 18.5532 15.3013C18.1844 14.9367 18 14.4776 18 13.9241C18 13.357 18.1844 12.8979 18.5532 12.5468C18.9362 12.1823 19.4184 12 20 12Z" fill="#F5F6FA"/>
                 </svg>
                 <div class="notification-wrap__text">
-                    <p>If the team member you want to invite to join the project is still not on this Satellite, please share this link to the signup page and ask them to register here: <a>www.storj.io/satellite/register</a></p>
+                    <p>If the team member you want to invite to join the project is still not on this Satellite, please share this link to the signup page and ask them to register here: <a v-on:click="redirectToRegister">{{registerPath}}</a></p>
```
Comment (egonelbre) [inline web/satellite/src/components/team/AddUserPopup.vue]: Any reason we aren't rendering the link to html directly?

**Pull request merged**